lua/lspcontainers/init.lua

```diff
@@ -67,12 +67,17 @@ local supported_languages = {
 }
 
 -- default command to run the lsp container
-local default_cmd = function (runtime, workdir, image, network)
+local default_cmd = function (runtime, workdir, image, network, docker_volume)
   if vim.fn.has("win32") then
     workdir = Dos2UnixSafePath(workdir)
   end
 
-  local volume = workdir..":"..workdir
+  local mnt_volume
+  if docker_volume ~= nil then
+    mnt_volume ="--volume="..docker_volume..":"..workdir
+  else
+    mnt_volume = "--volume="..workdir..":"..workdir
+  end
```

Comment:
So when mounting any volume, it'll automatically become the workdir?

I was thinking that volumes could be useful for things like global caches and alike, but I'd never want those to be the workdir.

Reply:
All my code get cloned into a workdir and because I have dockerized neovim and other dependencies then I mount in my docker volume with code.
So if I get different customers I have different volumes mounted instead of mount my host file system folder, maybe its a bad idea but I am trying to go the whole way using docker for the whole setup not only parts of it.

Reply:
Don't some LSPs expect the project to be in `pwd`? Or look for files relative to `pwd`?

Lemme pull this and check, maybe nothing breaks. I have a few LSPs, so might take a while to notice an infrequently used one.

Reply:
Yup, LSP's work fine. I'm impressed, I expected bugs :joy:

Reply:
I did this test - I promise lol